src/main/java/org/generationcp/middleware/dao/dms/DmsProjectDao.java

```diff
@@ -1372,4 +1385,85 @@ public DatasetDTO getDatasetOfSampleList(final Integer sampleListId) {
 		}
 	}
 
+	public List<MeasurementVariable> getEnvironmentConditionVariablesByGeoLocationIdAndVariableIds(Integer geolocationId, List<Integer> variableIds) {
```

Comment:
Move `getEnvironmentConditionVariablesByGeoLocationIdAndVariableIds `and `getEnvironmentDetailVariablesByGeoLocationIdAndVariableIds` methods to `GeolocationDao`